dockstore-client/src/main/java/io/dockstore/client/cli/YamlVerifyUtility.java

```diff
@@ -80,6 +81,15 @@ private static List<String> filesExist(List<String> paths, String base) {
         return missingFiles;
     }
 
+    // Determines if all the file paths are absolute
+    private static boolean checkAbsoluteFilePaths(List<String> paths) {
+        for (String path : paths) {
+            if (path.length() != 0 && !path.startsWith("/")) {
```

Comment:
Nit/optional, instead of `path.length() != 0`, you could have `!path.isEmpty()`. The implementation `isEmpty()` is your code, but `isEmpty()` arguably better conveys what you're trying to do.